widgy/models/versioning.py

```diff
@@ -193,9 +193,18 @@ def get_owner_related_names(cls):
 
     @cached_property
     def owners(self):
-        return list(owner
-                    for attr in self.get_owner_related_names()
-                    for owner in getattr(self, attr).all())
+        acc = []
+        for attr in self.get_owner_related_names():
+            manager = getattr(self, attr)
+
+            # If the manager has a get_widgy_owners() method, use this.
```

Comment:
This comment is obvious. Explain why you're doing what you're doing, not what you're doing.